src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+  - [Structured text (ontology term)](#structured-text-ontology-term)
+  - [URL](#url) (or other link utilising a URL like structure)
+  - [Mixed type (free text or ontology)](#mixed-type) (either free text, or from an ontology)
+
+You can use these as templates on how to construct your own new slot to the highest quality level expected.
+
+## Overview
+
+Every MIxS LinkML slot should have at a minimum the following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions): the description of what the metadata term is for
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/): a short human readable 'title' for the slot
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/): examples values demonstrating how the slot should be used
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/): the section of the schema that the slot belongs based on a [fix list of MIxS categories](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21-L26)
+  - Note: this system may be replaced in the near future!
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/): useful keywords to allow grouping of related slots together
+- [`slot_uri`](https://linkml.io/linkml-model/latest/docs/slot_uri/): a unique ID assigned by MIxS
+  - This likely only gets assigned upon acceptance and merging by the core GSC MIxS team
+  - During development you can use a dummy value for this
+- [`range`](https://linkml.io/linkml/schemas/slots.html#ranges): specifying the type of 'object' that put in the slot (numeric, string, etc.)
+
+And for some slots, the following attributes are also recommended:
+
+- If a term should be either mandatory or optional
+  - [`recommended`](https://linkml.io/linkml/schemas/slots.html#recommended): a boolean value indicating if the slot is recommended be filled
+  - [`required`](https://linkml.io/linkml/schemas/slots.html#required): a boolean value indicating if the slot is mandatory to be filled
+- If a term should be allowed multiple entries
+
+  - [`multivalued`](https://linkml.io/linkml/schemas/slots.html#multivalued)
+    - Essentially indicates the contents of the slot can be a list, and each element is evaluated independently against the remaining slot attributes
+
+- If a string or mix-`range`-based term has a particular format it should follow:
+
+  - [structured_pattern](https://linkml.io/linkml/schemas/constraints.html#structured-patterns): a particular regex-like pattern that includes pre-defined components that describe how each component should be formatted
+    - In the MIxS LinkML schema, these preset formats can be seen under the [`settings`](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21852-L21887) section of the schema
```

Comment:
Hopefully our structured_patterns won't look regex-like! They could be described as using a domain-specific language, or you could say that they compose the settings like macros?

Reply:
Maybe I'm too young (or skipped my excel days), but I've never used/seen a macro :sweat_smile: but can update teh text accordingly :+1: